### PR TITLE
Add jsonLD documentation and schema Jsonld

### PR DESCRIPTION
**Pull request by mlabadie**
- context:
  - create jsonld file to create an ontology between json file 

- add files:
    - schema.jsonld: contain jsonld schema filed
    - documentation schema jsonld.md: contain documentation associate to jsonld file

This files is correct?

### CODE REVIEW

File: schema.jsonld [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+  "@context":{
+    
+    "schema":"http://www.schema.org/",
+    "codemeta":"https://codemeta.github.io/terms/",
+    
+    "Organization":{"@id":"schema:Organization"},
```
Comment (pradal) [inline schema.jsonld:7]: Organization -> organization (see VIPS)

Comment (pradal) [reply schema.jsonld:7]: languages is missing

Comment (pradal) [reply schema.jsonld:7]: Add also models (list of model)

File: schema.jsonld [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    "Person":{"@id":"schema:Person"},
```
Comment (pradal) [inline schema.jsonld:8]: What is Person? authors?

**Review by pradal — CHANGES_REQUESTED** [review]
The schema is not complete.
We need to define both a DSS schema, and a model schema.
Look at format/DSS_metadat/VIPS.yaml to have  the name of the fields

File: schema.jsonld (hunk 2 above)
Comment (mlabadie) [reply schema.jsonld:8]: Person is a type in schema allows to describe person information
authors can by define by Person type. My context is almost the same as in meta data code.

Comment (pradal) [reply schema.jsonld:8]: Yes. But here it is a field, not a type, isn't t?

**Pull request merged**